core/src/main/scala/cell/Cell.scala

```diff
@@ -73,6 +73,9 @@ trait Cell[K <: Key[V], V] {
   // Only used in tests.
   private[cell] def waitUntilNoNextDeps(): Unit
 
+  private[cell] def isRunning: Boolean
+  private[cell] def markAsRunning(): Boolean
```

Comment:
I'm not 100% happy with the terminology. A cell by itself is not running. Rather, there may be a computation running that is eventually going to update the cell's value. Perhaps something like "tasksActive"?

Reply:
Changed it. Still "setTasksActive" does not sound great. Suggestions are welcome.